tests/blueprints/blueprints-list.directive.spec.js

```diff
@@ -0,0 +1,692 @@
+                "invalid": false,
+                "action_order": 0,
+                "backgroundColor": "#fff",
+                "outputConnectors": []
+              }
+            ],
+            "connections": []
+          },
+          "automate_entrypoints": {
+            "Provision": "Service/Provisioning/StateMachines/ServiceProvision_Template/default",
+            "Reconfigure": "/Grandchild 1"
+          }
+        },
+        "created_at": "2016-09-26T17:05:08Z",
+        "updated_at": "2016-09-29T18:25:59Z",
+        "content": {},
+        "actions": [
+          {
+            "name": "edit",
+            "method": "post",
+            "href": "http://localhost:8001/api/blueprints/10000000000023"
+          },
+          {
+            "name": "delete",
+            "method": "post",
+            "href": "http://localhost:8001/api/blueprints/10000000000023"
+          },
+          {
+            "name": "publish",
+            "method": "post",
+            "href": "http://localhost:8001/api/blueprints/10000000000023"
+          },
+          {
+            "name": "delete",
+            "method": "delete",
+            "href": "http://localhost:8001/api/blueprints/10000000000023"
+          }
+        ]
+      },
+      {
+        "href": "http://localhost:8001/api/blueprints/10000000000026",
+        "id": 10000000000026,
+        "name": "Blueprint Three",
+        "description": "The Thirdblueprint",
+        "ui_properties": {
+          "num_items": 0,
+          "visibility": {
+            "id": 10000000000001,
+            "href": "http://localhost:8001/api/tenants/10000000000001",
+            "name": "Red Hat"
+          },
+          "service_catalog": {
+            "id": 10000000000007
+          },
+          "chart_data_model": {
+            "nodes": [],
+            "connections": []
+          },
+          "automate_entrypoints": {
+            "Provision": "Service/Provisioning/StateMachines/ServiceProvision_Template/default"
+          }
+        },
+        "created_at": "2016-09-29T18:44:36Z",
+        "updated_at": "2016-09-29T18:44:36Z",
+        "content": {},
+        "actions": [
+          {
+            "name": "edit",
+            "method": "post",
+            "href": "http://localhost:8001/api/blueprints/10000000000026"
+          },
+          {
+            "name": "delete",
+            "method": "post",
+            "href": "http://localhost:8001/api/blueprints/10000000000026"
+          },
+          {
+            "name": "publish",
+            "method": "post",
+            "href": "http://localhost:8001/api/blueprints/10000000000026"
+          },
+          {
+            "name": "delete",
+            "method": "delete",
+            "href": "http://localhost:8001/api/blueprints/10000000000026"
+          }
+        ]
+      }
+    ];
+  }
+
+  function getServiceCatalogs() {
+    return [{
+      "href": "http://localhost:8001/api/service_catalogs/10000000000001",
```

Comment:
I like the idea, but it looks like it would take a little more work to implement:

http://stackoverflow.com/questions/17370427/loading-a-mock-json-file-within-karmaangularjs-test